src/test/java/net/fabricmc/fabric/Run.java

```diff
@@ -0,0 +1,29 @@
+/*
```

Comment:
You surely didn't want to push this?

Reply:
Curses. I could have sworn I had gitignored this file.